yellow-paper/docs/decentralization/actors.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Nodes and Actors
+---
+
+To analyze the suitability of different node types for various actor types in your system, let's break down the node types and actor types, considering the characteristics, preferences, abilities, and potential motivations of each actor. This will help determine which actors are likely to run which nodes and how they might interact with the Private eXecution Environment (PXE).
+
+## Node Types
+
+The below node types are ordered in resource requirements. The types exists both in the context of Aztec but also other blockchains. All the nodes, participate in the peer-to-peer (p2p) network, but with varying capacity.
+
+1. **Light Node**: 
+    - Follows the blockchain but only interested in headers (stores headers).
+    - Minimal resource requirements.
+    - Can be used to anchor state membership proofs with hard-to-manipulate state root.
+2. **Pruned Full Node**:
+   - Stores complete state, but only subset of logs and history.
+   - Requires less storage than a Full Node.
+   - Can be used to simulate transactions and view current state.
+2. **Full Node**:
+    - Stores complete state, logs and history.
```

Comment:
Is the distinction between full and archive nodes:

- History is the contents of blocks and logs stored.
- Historical state is the ability to say leaf index was 'x' at block 'y' and provide corresponding hashpaths.